src/extension.ts

```diff
@@ -166,7 +166,13 @@ export function activate(context: vscode.ExtensionContext) {
     });
     initAsyncCommand(context, 'deploymentSlot.SwapSlots', async (node: DeploymentSlotNode) => {
         if (node) {
-            await node.swapDeploymentSlots(outputChannel);
+            try {
+                outputChannel.appendLine('Initializing deployment slot swap...');
+                await node.swapDeploymentSlots(outputChannel);
+            } catch (err) {
+                outputChannel.appendLine(err.message);
```

Comment:
This is a somewhat common pattern used in this extension - but I would prefer if generic errors bubbled up to be caught in our telemetry

Additionally - I feel errors should use `window.showErrorMessage()` instead of output channel, which might not be open